etc/scripts/install.sh

```diff
@@ -123,3 +123,14 @@ wait_for_all_pods knative-eventing
 wait_for_all_pods knative-serving
 
 oc get pods --all-namespaces
+
+# Make the internal registry usable for Knative serving by skipping tag resolving
+val=$(oc -n knative-serving get cm config-controller -ojson | jq -r .data.registriesSkippingTagResolving | awk '{print $1",docker-registry.default.svc:5000"}')
+oc -n knative-serving get cm config-controller -ojson | jq ".data.registriesSkippingTagResolving = $val" | oc apply -f -
+
+# Add Golang imagestreams to be able to build go based images
+oc import-image -n openshift golang --from=centos/go-toolset-7-centos7 --confirm
+oc import-image -n openshift golang:1.11 --from=centos/go-toolset-7-centos7 --confirm
+
+# Make Knative Build work
+oc policy add-role-to-user admin system:serviceaccount:myproject:default
```

Comment:
I'm slightly uncomfortable with this one and I'm debating removing it since it's namespace specific (`myproject`)